src/main/java/core/basesyntax/service/BalanceOperation.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.Inventory;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperation implements FruitOperationHandler {
+    @Override
+    public void getFruitOperation(FruitTransaction transaction, Inventory inventory) {
```

Comment:
According to the checklist, there should not be an empty line at the beginning of class or method implementations. Please remove the empty line before the method declaration.